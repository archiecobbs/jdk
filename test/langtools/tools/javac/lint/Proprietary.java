/*
 * @test /nodynamiccopyright/
 * @bug 8349848
 * @summary Test for -Xlint:proprietary
 *
 * @compile/fail/ref=Proprietary.1.out  -XDrawDiagnostics -Werror                     Proprietary.java
 * @compile/ref=Proprietary.2.out       -XDrawDiagnostics -Werror -Xlint:-proprietary Proprietary.java
 */

class Proprietary {
    sun.misc.Unsafe x;      // should get a warning here
    @SuppressWarnings("proprietary")
    sun.misc.Unsafe y;      // should not get a warning here
}
