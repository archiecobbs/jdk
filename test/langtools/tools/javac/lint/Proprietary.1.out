Proprietary.java:11:13: compiler.warn.sun.proprietary: sun.misc.Unsafe
- compiler.err.warnings.and.werror
1 error
1 warning
